### PR TITLE
Siterecording gtm

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+### Installation
+Integrating Site Recording to your website doesn’t require any coding with Google Tag manager. All you have to do is create a new tag using our template in the Google tag manager template Gallery.
+
+1. Log in to Google Tag Manager, navigate your account and click on "Add a new tag".
+
+2. Search for our community template "Site Recording by 500 apps " in the "Community Template Gallery".
+
+3. Add the template to your tag by clicking on "Add to Workspace".
+
+4. Copy the key from the application and place it in the "PushNinja Key" field in your tag after the template is added.
```
Comment (subramaniyamkolluri) [inline README.md]: Siterecording

File: metadata.yaml [hunk 2]
```diff
@@ -1,9 +1,9 @@
-homepage: "https://pushninja.com"
+homepage: "https://siterecording.com"
 documentation: ""
 versions:
   # Latest version
-  - sha: 4d93eb730943f4b3d508a1d83202363a4f0776dd
-    changeNotes: Added the helptext and changed the label.
+  - sha: aa99cc0892da261c017266a540655f1e2884be0b
+    changeNotes: Updated the template file.
```
Comment (subramaniyamkolluri) [inline metadata.yaml]: First Release

File: metadata.yaml [hunk 3]
```diff
@@ -1,9 +1,6 @@
...
 versions:
-  # Latest version
+  # First Release
   - sha: aa99cc0892da261c017266a540655f1e2884be0b
-    changeNotes: Updated the template file.
-  # Older versions
-  - sha: 9671822bd1a72946803f671ddc8402e571ba30b6
-    changeNotes: Fixed the gtm url.
\ No newline at end of file
+    changeNotes: Updated the template file.
```
Comment (subramaniyamkolluri) [inline metadata.yaml]: Notes too

File: README.md [hunk 4]
```diff
@@ -1 +1,39 @@
-# PushNinja
+# Site Recording by 500apps - Website Visitor Tracker 
+Site recording is a tool that records browsing sessions of website visitors. It enables marketers to understand the entire user journeys on the website as well as how they use your site, allowing you to see everything they do. Examine each mouse movement, scroll, click on your website.
```
Comment (agilecrm) [inline README.md]: Butler English

File: template.tpl [hunk 5]
```diff
@@ -14,13 +14,13 @@ ___INFO___
...
+    "thumbnail": "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADYAAAAyCAYAAAAX1CjLAAACU0lEQVRoBe2Yv07DMBDG+048ADwAL8DCxp+5rEhsnUCCscyUESRYYCqCtZ0rRgYWIii0lKKCMPqQLnKusWts4rjBJ1mJkzj2777zxXFNVNRqFeUSEWzelI2KRcUC8UAMRZUQg9GXOL8Zi+bpyEu56r6rhpK57qRYpzcRS5uJWFh98FpWtp/EffKZAeEVa7Dbu48UanEjEfX9Z9E8eS20rDX6qQMBpzNrMIBAqeX6o0A4+jI4lCKkdfGm7NYajF5+dj1WvryoGzuHgx84OFdlzmCYZ74NIQ/HIjRVFsG4ZygUK6sYAZZxLDQUywCiPgsFq2woRjCefRzqMd3bOI8m8L8KRawdtw5exHqjb1x+szQrLRShIilqesRgyfBvh8WuykoDw4Cw8sYATIsc0nAG/vOOL/NX76WCqbxtcl1WGSHNf4sqAUbqyYri7xmrjrlbeciKyed7rWFGPRmWR0Jwvy1QQ4bh59gS0CUWAvQOhvmR9xmgdG+aUVWJpTQwbPxwFVAHMMwUDG12j4bEMXX0rhhCLe9TQPNFB4aNIyQM7HnAEbqQ9A425Vp2QQeGbxtP/ax5Wg0ejLb5KHwpZFMCxYkzWLtjtuWs6H/qsqwY7RsiExKYqWrWYNQZMtxfGm2IYh6RybCzkga1sQaDUuRFwCH9dnsT6yLvxQOEG5IG9Yej/Dx/FnVrMDRGvMuduZzPmjtcNV1GdAbDC9AhwobWbrZH+kDjnSoDPJJJnqK8jZNi/GUh1SNYSGqYjCUqZuKlkJ6JioWkhslYomImXgrpmcoq9g1GpLaRi5mIlQAAAABJRU5ErkJggg\u003d\u003d"
   },
-  "description": "Send Push Notifications like a Ninja.",
+  "description": "Site Recording - Visitor Tracking Software",
```
Comment (agilecrm) [inline template.tpl]: Site Recording - Session Recording Software

**Comment by agilecrm** [comment]
pl tag me once the changes are completed

File: metadata.yaml (hunk 3 above)
Comment (subramaniyamkolluri) [reply metadata.yaml]: This is our first release. give right content in changeNotes

 not older/latest release

File: README.md [hunk 6]
```diff
@@ -14,7 +13,7 @@ Integrating Site Recording to your website doesn’t require any coding with Goo
 
 3. Add the template to your tag by clicking on "Add to Workspace".
 
-4. Copy the key from the application and place it in the "Siterecording Key" field in your tag after the template is added.
+4. Copy the key from the application and place it in the "Site Recording Key" field in your tag after the template is added.
```
Comment (subramaniyamkolluri) [inline README.md]: is it SiteRecording or Site Recording or Siterecording

File: README.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ Integrating Site Recording to your website doesn’t require any coding with Goo
 
 1. Log in to Google Tag Manager, navigate your account and click on "Add a new tag".
 
-2. Search for our community template "Site Recording by 500 apps " in the "Community Template Gallery".
+2. Search for our community template "Site Recording by 500 apps" in the "Community Template Gallery".
```
Comment (subramaniyamkolluri) [inline README.md]: Is it Site Recording or Siterecording or SiteRecording

check with website and marketing team too

File: README.md (hunk 6 above)
Comment (shiva-500apps) [reply README.md]: Sir, Its SiteRecording.

File: README.md (hunk 7 above)
Comment (shiva-500apps) [reply README.md]: Sir, Its SiteRecording.

File: metadata.yaml (hunk 3 above)
Comment (ghost) [reply metadata.yaml]: Changed

File: template.tpl [hunk 8]
```diff
@@ -33,14 +33,14 @@ ___TEMPLATE_PARAMETERS___
...
-    "displayName": "PushNinja Key",
+    "displayName": "SiteRecording Key",
     "simpleValueType": true,
     "valueValidators": [
       {
         "type": "NON_EMPTY"
       }
     ],
-    "help": "Please copy the PushNinja Key from the share script sidebar in the application."
+    "help": "Please copy the SiteRecording Key from Website -\u003e Share/Copy script sidebar -\u003e Connections -\u003e Google Tag Manager at https://infinity.500apps.com/siterecording"
```
Comment (agilecrm) [inline template.tpl:43]: What is this special character? \u003?

Comment (shiva-500apps) [reply template.tpl:43]: sir, its encoded as we export the file from "->" to \u003e

**Pull request merged**